### PR TITLE
Fleet bundle details UI update - #4793

### PR DESCRIPTION
**Pull request by Shavindra**
## Summary

Issue #4793  - Fleet bundle details UI update

### CODE REVIEW

File: components/formatter/FleetSummaryGraph.vue [hunk 1]
```diff
@@ -20,7 +20,7 @@ export default {
     },
 
     show() {
-      return this.stateParts.length > 0 && this.row.targetClusters?.length;
+      return this.stateParts.length > 0; // && this.row.targetClusters?.length;
```
Comment (aalves08) [inline components/formatter/FleetSummaryGraph.vue:23]: To be cleared, I believe

File: detail/fleet.cattle.io.gitrepo.vue [hunk 2]
```diff
@@ -22,7 +26,49 @@ export default {
...
+          return !harvester[targets[0].clusterName];
+        }
+
+        return true;
+      });
+
+      return bundles;
+    },
+  },
   async fetch() {
```
Comment (aalves08) [inline detail/fleet.cattle.io.gitrepo.vue:69]: If these dispatches/requests don't have a requisite to be in series, you should consider doing them all in parallel using `await allHash`. Will improve load time

https://github.com/rancher/dashboard/blob/master/pages/c/_cluster/fleet/index.vue#L19

**Review by aalves08 — CHANGES_REQUESTED** [review]
Left some comments @Shavindra . Let me know if you need to go through them together. Overall, good work! Congrats

File: components/fleet/FleetBundles.vue [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+  },
+
+  computed: {
+
+    schema() {
+      return this.$store.getters['management/schemaFor']( FLEET.BUNDLE );
+    },
+
+    repoName() {
+      return this.value.metadata.name;
```
[the remark this answers is not in the document]
Comment (Shavindra) [reply components/fleet/FleetBundles.vue:49]: This is always present. So it should be OK

File: components/fleet/FleetBundles.vue [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+
+  props: {
+    value: {
+      type:     Object,
+      required: true,
+    },
+  },
+
+  async fetch() {
+    this.allBundles = await this.$store.dispatch('management/findAll', { type: FLEET.BUNDLE });
```
[the remark this answers is not in the document]
Comment (Shavindra) [reply components/fleet/FleetBundles.vue]: Fixed

File: components/fleet/FleetResources.vue [hunk 5]
```diff
@@ -123,13 +125,16 @@ export default {
...
-    :rows="computedResources"
-    :headers="resourceHeaders"
-    :table-actions="false"
-    :row-actions="false"
-    key-field="key"
-    default-sort-by="state"
-    :paged="true"
-  />
+  <div>
+    {{ JSON.stringify(value) }}
```
[the remark this answers is not in the document]
Comment (Shavindra) [reply components/fleet/FleetResources.vue]: Fixed

File: components/formatter/FleetSummaryGraph.vue (hunk 1 above)
Comment (Shavindra) [reply components/formatter/FleetSummaryGraph.vue]: Fixed

File: detail/fleet.cattle.io.gitrepo.vue (hunk 2 above)
Comment (Shavindra) [reply detail/fleet.cattle.io.gitrepo.vue:76]: This is fixed I am not sure why its not showing up as outdated.

**Review by aalves08 — APPROVED** [review]

**Pull request merged**